packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1565,12 +1573,13 @@ export class ColorPicker
 
     const x = alphaToOpacity(hsvColor.alpha()) / (OPACITY_LIMITS.max / width);
     const y = radius;
+    const sliderBoundX = remap(x, 0, width, radius, width - radius);
```

Comment:
This will cause the thumb to not center on `pointerup` event. `sliderBoundX` is remapped on to a new range which is adjusting the thumb to move away from the center. Applies to opacity slider also.

Reply:
I'm not following. Both thumbs and scopes are centered. 🤔

One thing to note is that the scopes will look misaligned in non-HD screens (this is currently how it's rendering on `@next`). Is this what you're observing?

### HD

![Screenshot 2023-08-02 at 1 28 27 PM](https://github.com/Esri/calcite-design-system/assets/197440/5cc37e6c-7f26-4405-aacd-ee3bde541adb)
![Screenshot 2023-08-02 at 1 28 31 PM](https://github.com/Esri/calcite-design-system/assets/197440/51a89050-420b-4b80-bcdc-874206f178c9)
![Screenshot 2023-08-02 at 1 28 34 PM](https://github.com/Esri/calcite-design-system/assets/197440/0355a448-b6d7-4871-8988-01dbb44b22dd)

### Standard

![Screenshot 2023-08-02 at 1 29 10 PM](https://github.com/Esri/calcite-design-system/assets/197440/344a8b95-9469-430d-9845-77ca6ae21b7c)
![Screenshot 2023-08-02 at 1 29 14 PM](https://github.com/Esri/calcite-design-system/assets/197440/278e0ceb-1fe8-4782-8894-a2f55b875f85)
![Screenshot 2023-08-02 at 1 29 17 PM](https://github.com/Esri/calcite-design-system/assets/197440/880e5dd1-e95c-4c0c-9031-15908b6ddbfe)

Reply:
Confirmed w/ @anveshmekala that there's an issue when sampling colors individually vs sliding. Thanks for clarifying!

I'll look into only mapping if the x lies within the slider edges.